gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/initialize.flow.ts

```diff
@@ -35,8 +50,15 @@ export default class OCR2InitializeFlow extends FlowCommand<TransactionResponse>
         id: this.stepIds.OCR_2,
       },
       {
-        name: 'Change RDD',
-        exec: this.showRddInstructions,
+        name: 'Change RDD Identifier',
+        exec: this.showRddIdentifierInstructions,
+      },
+      {
+        name: 'Set writer on Store',
```

Comment:
@aalu1418 This is run right after the OCR2 feed is initialized. I think is correct?

Reply:
I think that should work. all you need is the store account (from the initialize store), transmissions account (from the create feed), and storeAuthority (calculated in the ocr2 initialize)